syntax = "proto3";

package tutorial;

import "my_proto/address.proto";

message NameQuery {
    string name = 1;
}

service AddressBook {
    rpc GetAddress(NameQuery) returns (Address) {}
}

